From owner-linux-pci@atrey.karlin.mff.cuni.cz Thu Nov 30 15:59:15 2006
From: Inaky Perez-Gonzalez <inaky@linux.intel.com>
To: Greg KH <greg@kroah.com>
Subject: pci: add class codes for Wireless RF controllers
Date: Thu, 30 Nov 2006 15:58:58 -0800
Cc: linux-pci@atrey.karlin.mff.cuni.cz
MIME-Version: 1.0
Message-Id: <200611301558.58473.inaky.perez-gonzalez@intel.com>

pci: add class codes for Wireless RF controllers

Add PCI codes to include/linux/pci_ids.h for RF controllers; first
batch of these devices seem to be the Ultra-Wide-Band and Wireless USB
controllers (WHCI spec).

Signed-off-by: Inaky Perez-Gonzalez <inaky@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/pci_ids.h |    4 ++++
 1 file changed, 4 insertions(+)

--- gregkh-2.6.orig/include/linux/pci_ids.h
+++ gregkh-2.6/include/linux/pci_ids.h
@@ -104,6 +104,10 @@
 #define PCI_CLASS_SERIAL_FIBER		0x0c04
 #define PCI_CLASS_SERIAL_SMBUS		0x0c05
 
+#define PCI_BASE_CLASS_WIRELESS			0x0d
+#define PCI_CLASS_WIRELESS_RF_CONTROLLER	0x0d10
+#define PCI_CLASS_WIRELESS_WHCI			0x0d1010
+
 #define PCI_BASE_CLASS_INTELLIGENT	0x0e
 #define PCI_CLASS_INTELLIGENT_I2O	0x0e00
 
